news/non-contiguous-arrays.rst

```diff
@@ -0,0 +1,30 @@
+New features
+------------
+
+* <news item>
+
+Deprecations
+------------
+
+* <news item>
+
+Exposing HDF5 functions
+-----------------------
+
+* <news item>
+
+Bug fixes
+---------
+
+* Correctly store non-contiguous arrays, such as numpy slices.
```

Comment:
Specifically, it's storing them as HDF5 variable length data that was failing before.

Otherwise, this looks good, thanks.

Reply:
Thanks, I've amended the comment.

Reply:
Did something go wrong with committing it? The text I can see on Github still looks the same as before.

Reply:
I forgot to do got add, yes